packages/react-server/core/renderMiddleware.js

```diff
@@ -494,8 +483,18 @@ function renderScriptsAsync(scripts, res) {
 		// This is the full implementation of LABjs.
 		res.write(LABString);
 
-		// We always want scripts to be executed in order.
-		res.write("$LAB.setGlobalDefaults({AlwaysPreserveOrder:true});");
+		if (process.env.NODE_ENV === "test") { // eslint-disable-line no-process-env
+			// ZombieJS does not handle loading async scripts in the proper order and ensuring that .wait() waits for
+			// execution.  Thus, we need to force LABjs to not use local XHRs in order for all tests to work properly.
+			// This used to work prior to combining the HTML and JS servers into a single server because the JS files
+			// were considered part of a different domain when running tests, so it never used XHR preloading, it used
+			// "cache preloading".
+			// https://github.com/getify/LABjs/blob/master/LAB.src.js#L152-L175
+			res.write("$LAB.setGlobalDefaults({UseLocalXHR:false,AlwaysPreserveOrder:true});");
+		} else {
+			// We always want scripts to be executed in order.
+			res.write("$LAB.setGlobalDefaults({AlwaysPreserveOrder:true});");
+		}
```

Comment:
LABjs allows us to load the webpack scripts themselves at page load as async as possible in all browsers to prevent blocking paint, etc.